Clew/Assets.xcassets/Align.imageset/Contents.json

```diff
@@ -2,17 +2,17 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "Align.png",
+      "filename" : "landmark2.png",
       "scale" : "1x"
     },
     {
       "idiom" : "universal",
-      "filename" : "align@2x.png",
+      "filename" : "landmark2@2x.png",
```

Comment:
Is this is just renaming for consistency?

Reply:
this actually changes the file form the carpenter's square image to a thumbtack image